languages/java/oso/src/main/java/com/osohq/oso/Host.java

```diff
@@ -167,15 +167,10 @@ public boolean isSubclass(String leftTag, String rightTag) {
     return rightClass.isAssignableFrom(leftClass);
   }
 
-  /** Check if two instances unify. */
-  public boolean unify(long leftId, long rightId) throws Exceptions.UnregisteredInstanceError {
-    Object left = getInstance(leftId);
-    Object right = getInstance(rightId);
-    if (left == null) {
-      return right == null;
-    } else {
-      return left.equals(right);
-    }
+  public boolean operator(String op, List<Object> initargs) throws Exceptions.OsoException {
```

Comment:
Why is this called `initargs` rather than just `args` like other languages?

Reply:
I was trying to be consistent with `makeExternal` but in hindsight it's obviously short for "initializer args" which makes no sense here. Changed it :)

Reply:
Ohh I didn't see `makeExternal`.